### PR TITLE
30 add login page client side input validation

### PR DESCRIPTION
**Pull request by rhysmah**
## Summary

Added client-side validation using `Yup` and `Formik`, identical to what's used on `RegistrationForm`.

## Testing

1. Navigate to the landing page using Deploy Preview (below)
2. Navigate to the `Login` page
3. Click `Submit` and confirm that the `email` and `password` fields outline in red and display error messages
4. Fill in the `email`, don't fill in `password`, click `Submit`, and confirm the password field goes red and displays an error
5. Fill in the `password`, don't fill in `email`, click `Submit`, and confirm the email field goes red and displays an error
6. Fill in both fields with valid inputs and click the `Submit`; confirm, in Google Developer Tools, that an object is created containing both the entered email and password.

### CODE REVIEW

File: apps/frontend/src/components/LoginForm.tsx [hunk 1]
```diff
@@ -1,26 +1,124 @@
...
+          </Button>
+        </HStack>
+        <Button
+          type="submit"
+          w={'100%'}
+          colorScheme="brand"
+          variant="solid"
+          disabled={formik.isSubmitting}
+        >
+          SIGN UP
```
Comment (toco-t) [inline apps/frontend/src/components/LoginForm.tsx:117]: It should probably be `LOG IN`...?

**Review by toco-t — CHANGES_REQUESTED** [review]
Everything looks great except for a minor typo. Well done!

Also, I've just merged some changes so please fetch them :)

**Comment by rhysmah** [comment]
I fetched your changes and rebased  this branch. Made your change. 

Let me know everything's okay, or if there are any issues. Once this is resolved, I have a PR for #74, and I'd love to get your feedback on that.

**Review by toco-t — APPROVED** [review]

**Comment by toco-t** [comment]
It seems that the deployment failed after you pushed the last change. You should check the logs before merging your branch!

**Pull request merged**